projects/website/src/app/news/news.component.ts

```diff
@@ -29,6 +29,8 @@ export class NewsComponent implements OnDestroy, AfterViewInit {
   @ViewChildren(BugFix) bugFixes: QueryList<BugFix>;
   @ViewChildren(NewComponent) newComponents: QueryList<NewComponent>;
 
+  movedToGithub: boolean;
```

Comment:
Don't think we need to have `movedToGithub` boolean - we could just add a template with the content of `projects/website/src/releases/12.x/12.0.13-github.html` 

but change it a little bit:

```
<h2>Release Notes have moved to github!</h2>

<p>
  Release Notes for version 12.0.13 are now on
  <a href="https://github.com/vmware-clarity/ng-clarity/releases/tag/v13.0.3">GitHub Release</a>.
</p>
```

Reply:
I'm sorry, I'm not sure I follow what you mean by adding a new template?

Reply:
Going with multiple files for every version you could directly address a release - that is the change in the above example.

Provide a better and shareable link addressing an exact version. 

Because we gonna start getting questions like "where are the changes for version 13.0.4?" - and the reason for that is that for the past 4 years we have done for every release single page.

- I know that we gonna need to update this for every release - but I don't think that this is a lot of work - we did it in the past, now is even faster because you need to only create a link to the changelog and no need of digging inside issues and descriptions.

Also, suspect that this will be for a short period of time. When updating and picking one of the sites all this will be dropped and moved inside the "website that we pick" - so I hope that this will be resolved in the next 1-2 releases.

If we decide that we want to update it only once. Then there is a need to reformat the message to something else and not include `version+` notation - that could lead to people asking more questions :) 

If we go with one update only then something like that I think is more suitable:

```html
<h2>Release Notes have moved to Github!</h2>

<p>
  Release Notes for version above 12.0.x are now on part of
  <a href="https://github.com/vmware-clarity/ng-clarity/releases">GitHub Release</a>. 
</p>
```

And the navigation link must be something like `12.0.x`

Reply:
>Because we gonna start getting questions like "where are the changes for version 13.0.4?" - and the reason for that is that for the past 4 years we have done for every release single page.

Right, which is why the website now says "release notes are now on github" - just because we've done it that way for the last 4 years doesn't mean we have to continue to do it that way, at least until we have a long term plan for the website(s). 

We have this beautiful CI/CD solution the releases a new version in 7 minutes, it just feels so unnecessary to keep this manual process around. Just to add a link to github every release.

I'm happy to update the messaging and go with the X syntax instead, since the link will display all v12 releases that we have on github.

Reply:
Ah, I tried changing it to 12.0.x and the sort places it in the middle of the list :(

<img width="219" alt="image" src="https://user-images.githubusercontent.com/9469374/160417410-4dd758fc-26a8-4c06-90af-2962a527773c.png">